src/planning.ts

```diff
@@ -37,6 +38,7 @@ export interface PlanOptions {
 export const defaultPlanOptions: PlanOptions = {
   penUpHeight: 50,
   penDownHeight: 60,
+  penStrokeWidth: 0.5,
```

Comment:
It looks like this is now bigger than the previous default—before we had a `strokeWidth` of 0.5, but now we have a default `strokeWidth` of `0.5 * stepsPerMm`, which is 2.5. Should this default to 0.1 to keep it the same as before?

Reply:
> before we had a strokeWidth of 0.5

0.5 is for pen-up lines. There was no default for pen-down lines, so it defaulted to 1px (at least in Firefox where I checked). 1px equals 0.2mm. So to keep it what it was before it should be 0.2.

I thought it would be good to choose a common pen width instead of going back to what it was before. I chose 0.5 for that reason. Maybe a bit thinner might be good though. But I don't mind changing it to whatever you prefer.

Hope that makes sense. 🙂

Reply:
ahhh, gotcha. thanks for the clarification!